testutil/compose/auto.go

```diff
@@ -187,7 +187,7 @@ func fixPerms(ctx context.Context, dir string) error {
 func execDown(ctx context.Context, dir string) error {
 	log.Info(ctx, "Executing docker-compose down")
```

Comment:
For completeness' sake, rename all instances of `docker-compose` to `docker compose` please.

Reply:
Done!